src/content/sanitaetsdienst/sanitaeter.svx

```diff
@@ -42,39 +31,49 @@ Konkret können sie:
   - Herzstillstände behandeln
 - Brüche operieren
 
-## Verbrauchsmaterialien
+## Ausrüstung
+
+### Verbrauchsmaterialien
 
 Die folgenden Materialien sind zum einmaligen Gebrauch bestimmt.
 
 #### AAT-Kit
 
-gebraucht zur [Behandlung von Hämothoraxen und Spannungspneumothoraxen]().
+gebraucht zur [Behandlung von Lungenverletzungen](/articles/sanitaetsdienst/atmung#lungen).
 
 #### Field Blood Transfusion Kit (dt. Feldtransfusionsset)
 
-erlaubt es, im Feld Blut abzunehmen und Bedürftigen zu verabreichen. Siehe dazu [Transfusionen im Feld]().
+erlaubt es, im Feld Blut abzunehmen und Bedürftigen zu verabreichen. Siehe dazu [Transfusionen im Feld](/articles/sanitaetsdienst/blutkreislauf#transfusionen-im-feld).
 
 #### Chest Seal (dt. Thoraxverschlusspflaster)
 
-ein großes Pflaster mit einem Ein-Weg-Ventil. Das Ventil lässt Luft und Flüssigkeiten aus dem Brustkorb entweichen und verhindert so einen Pneumothorax, siehe [Behandlung]() von Hämothoraxen und Spannungspneumothoraxen.
+ein großes Pflaster mit einem Ein-Weg-Ventil. Das Ventil lässt Luft und Flüssigkeiten aus dem Brustkorb entweichen und verhindert so einen Pneumothorax, siehe [Behandlung von Lungenverletzungen](/articles/sanitaetsdienst/atmung#lungen).
 
 #### 16g IV
 
 eine 16g-Kanüle (Durchmesser 1,5 mm) für intravenöse (IV) Zugänge. Wird benötigt, um Flüssigkeiten wie Blutinfusionen, aber auch manche Medikationen zu verabreichen. Diese Nadeln können nicht wiederverwendet werden, wie [in diesem YouTube-Video](https://youtu.be/rzLuZWWZ2eQ) anschaulich gezeigt wird.
 
 #### FAST IO
 
-_First Access for Shock and Trauma IO,_ ein intraossärer (IO) Zugang am Brustbein. Wird verwendet, wenn die Extremitäten für einen intravenösen Zugang zu stark verletzt sind.
+_First Access for Shock and Trauma IO,_ ein intraossärer (IO) Zugang am Brustbein. Wird verwendet, wenn die Extremitäten für einen intravenösen Zugang zu stark verletzt sind. Außerdem ist kein Puls nötig, um über diesen Zugang Flüssigkeiten verabreichen zu können<sup>[aktuell?]</sup>.
```

Comment:
@LeonTReal I once read that FAST IOs can do that but I can't recall where and have never tried it out, we should probably confirm that.

Reply:
Tested in game. FAST IOs allow fluids to flow into the body without a heartbeat.